electron-wrapper.js

```diff
@@ -39,18 +39,18 @@ function open () {
     resizable: true
   }
 
-  w = new BrowserWindow(
-    bw_options
+  app.window = new BrowserWindow(
```

Comment:
Changes to allow mocha-electron to launch the electron app. Testing the UI in electron is debatable at this time as the UI is drawn on canvas.